### PR TITLE
Header style

### PR DESCRIPTION
**Pull request by Shazeen15**

### CODE REVIEW

File: app/assets/stylesheets/landing.scss [hunk 1]
```diff
@@ -1,11 +1,14 @@
 // #daf2dc62
+
```
Comment (sjproctor) [inline app/assets/stylesheets/landing.scss]: This can be removed.

File: app/javascript/components/components/LandingPage.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React, { Component } from "react";
-// import video1 from "../../../assets/videos/video1.mp4";
+// import video1 from "../assets/video1.mp4";
```
Comment (sjproctor) [inline app/javascript/components/components/LandingPage.js]: Is this being used?

**Review by sjproctor — APPROVED** [review]
Just a couple small things.

File: app/javascript/components/components/LandingPage.js (hunk 2 above)
Comment (Shazeen15) [reply app/javascript/components/components/LandingPage.js]: need to remove

**Pull request merged**